README.md

```diff
@@ -1,283 +1,53 @@
-*Note:* In preparation of `v3` of HighFive, we've started merging breaking
-changes into the main branch. More information and opportunity to comment can
-be found at:
-https://github.com/BlueBrain/HighFive/issues/864
+> [!WARNING]
+> The BlueBrainProject closed at the end of 2024. This unfortunately required
+> archiving all repositories under `BlueBrain` including HighFive.
```

Comment:
```suggestion
> The BlueBrainProject closed at the end of 2024. This unfortunately required
> ceasing development of all repositories under `BlueBrain` including HighFive.
```
who knows if these things are actually archived, or will just be left to rot.

Reply:
We've been told they'll be made read-only. I'd prefer not to write that we "cease development", the point is that it's moving homes, not stopping.

Reply:
It says "under `BlueBrain`" right after cease...